packages/test-generator/lib/generators/NodeTestGenerator.ts

```diff
@@ -44,6 +46,8 @@ export class NodeTestGenerator extends TestGenerator {
 
   private readonly formRendererFactory: any;
```

Comment:
Why are these `any`?

Reply:
These can be updated to something like this for views. I can update on a follow up. I think it's just leftover from an earlier time and can be updated now.
```
private readonly viewRendererFactory: StudioTemplateRendererFactory<unknown, StudioView, FrameworkOutputManager<unknown>, RenderTextComponentResponse, AmplifyViewRenderer>;
```